<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1102</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget {
    background-color: #FFF0F3; /* fond rose clair */
}

QLabel#labelWebcam,
QLabel#labelImage,
QLabel#labelVideo {
    border: 2px solid #A92B48;
    border-radius: 12px;
    padding: 8px;
}

QLabel#labelWebcam:hover,
QLabel#labelImage:hover,
QLabel#labelVideo:hover {
    background-color: #FFDCE2;
    cursor: pointer;
}
</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="QWidget" name="page_1_Select">
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>70</y>
          <width>965</width>
          <height>391</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="layoutMain">
         <item>
          <widget class="QLabel" name="label">
           <property name="palette">
            <palette>
             <active>
              <colorrole role="WindowText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Button">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>240</green>
                 <blue>243</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Text">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="ButtonText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>240</green>
                 <blue>243</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Window">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>240</green>
                 <blue>243</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="PlaceholderText">
               <brush brushstyle="SolidPattern">
                <color alpha="128">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
             </active>
             <inactive>
              <colorrole role="WindowText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Button">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>240</green>
                 <blue>243</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Text">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="ButtonText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>240</green>
                 <blue>243</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Window">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>240</green>
                 <blue>243</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="PlaceholderText">
               <brush brushstyle="SolidPattern">
                <color alpha="128">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
             </inactive>
             <disabled>
              <colorrole role="WindowText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Button">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>240</green>
                 <blue>243</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Text">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="ButtonText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>240</green>
                 <blue>243</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Window">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>240</green>
                 <blue>243</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="PlaceholderText">
               <brush brushstyle="SolidPattern">
                <color alpha="128">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
             </disabled>
            </palette>
           </property>
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>14</pointsize>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel {
    color: black;
    font: bold 14pt &quot;Segoe UI&quot;;
}

</string>
           </property>
           <property name="text">
            <string>Veuillez choisir un mode d’acquisition :</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>30</number>
           </property>
           <item>
            <widget class="ClickableLabel" name="labelVideo">
             <property name="pixmap">
              <pixmap resource="resources.qrc">:/images/images/video.png</pixmap>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="ClickableLabel" name="labelImage">
             <property name="pixmap">
              <pixmap resource="resources.qrc">:/images/images/image.png</pixmap>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="ClickableLabel" name="labelWebcam">
             <property name="pixmap">
              <pixmap resource="resources.qrc">:/images/images/webcam.png</pixmap>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2_Display">
       <property name="styleSheet">
        <string notr="true">/* ======== PANNEAU GAUCHE ======== */
QFrame#panneauGauche {
    background-color: #FFEFF4;
    border: 2px solid #B30033;
    border-radius: 22px; /* angles bien arrondis */
    padding: 15px;
}

/* Titre &quot;Mes traitements&quot; */
QLabel#labelTitreTraitements {
    color: #B30033;
    font-size: 20px;
    font-weight: 600;
    qproperty-alignment: AlignHCenter;
    font-family: &quot;Segoe UI&quot;, &quot;Arial&quot;, sans-serif;
}

/* ===== LISTE ===== */
QListWidget#listeTraitements {
    background-color: white;
    border: 2px solid #B30033;
    border-radius: 18px;   /* angles courbés */
    padding: 8px;
    font-size: 14px;
    font-family: &quot;Segoe UI&quot;, &quot;Arial&quot;, sans-serif;
    color: #333;
}

QListWidget::item {
    padding: 10px;
    margin: 6px 4px;
    border-radius: 12px;  /* arrondis pour les items */
}

QListWidget::item:hover {
    background-color: #FFE1EA;
    border-radius: 12px;
}

QListWidget::item:selected {
    background-color: #F7B6C4;
    border-radius: 12px;
    color: #000;
}

/* ======== PANNEAU CENTRE ======== */
QFrame#panneauCentre {
    background-color: #FFF7FB;
    border: 2px solid #B30033;
    border-radius: 22px;
    padding: 15px;
}

/* ======== PANNEAU DROITE ======== */
QFrame#panneauDroite {
    background-color: #FFF7FB;
    border: 2px solid #B30033;
    border-radius: 22px;
    padding: 15px;
}

/* ======== BOUTONS ======== */
QPushButton {
    background-color: #B30033;
    color: white;
    border-radius: 16px;  /* arrondi */
    padding: 10px 18px;
    border: 2px solid #8A0024;
    font-size: 14px;
    font-weight: 500;
    font-family: &quot;Segoe UI&quot;, &quot;Arial&quot;, sans-serif;
}

QPushButton:hover {
    background-color: #D31545;
}

QPushButton:pressed {
    background-color: #8A0024;
}
</string>
       </property>
       <widget class="QLabel" name="labelPreview">
        <property name="geometry">
         <rect>
          <x>860</x>
          <y>450</y>
          <width>121</width>
          <height>101</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QFrame" name="panneauGauche">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>191</width>
          <height>511</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <widget class="QWidget" name="verticalLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>-1</x>
           <y>-1</y>
           <width>191</width>
           <height>521</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="dispositionPanneauGauche">
          <item>
           <widget class="QWidget" name="zoneEtiquettes" native="true"/>
          </item>
          <item>
           <widget class="QLabel" name="labelTitreTraitements">
            <property name="text">
             <string>Mes traitements</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignHCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QFrame" name="panneauCentre">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>20</y>
          <width>621</width>
          <height>511</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <widget class="QWidget" name="verticalLayoutWidget_4">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>621</width>
           <height>511</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="dispositionPanneauCentre"/>
        </widget>
       </widget>
       <widget class="QFrame" name="panneauDroite">
        <property name="geometry">
         <rect>
          <x>880</x>
          <y>10</y>
          <width>221</width>
          <height>511</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <widget class="QWidget" name="verticalLayoutWidget_3">
         <property name="geometry">
          <rect>
           <x>-10</x>
           <y>110</y>
           <width>221</width>
           <height>511</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="dispositionPanneauDroite"/>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1102</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuVisionBox">
    <property name="title">
     <string>VisionBox</string>
    </property>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuVisionBox"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickableLabel</class>
   <extends>QLabel</extends>
   <header>clickablelabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
